README.md

```diff
@@ -67,3 +68,77 @@ automatically starts.
 It is very important that the commit that lands on **master** follows
 the conventional commit format, since that is what is used to
 automatically determine the next version.
+
+**Never push straight to master, always go through a PR!**
```

Comment:
We should be able to lock master so that no one can push directly anyways.
Is there a reason why we're not using this feature?

Reply:
The @dhis2-bot needs to push to master when it cuts the release, that's why it's not locked. I do think it's locked for non-admins though. This is just there for clarity.